src/presign/round_one.rs

```diff
@@ -19,11 +19,11 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Serialize, Deserialize)]
 pub(crate) struct Private {
     pub k: BigNumber,
-    pub rho: PaillierNonce,
+    pub rho: Nonce,
     pub gamma: BigNumber,
-    pub nu: PaillierNonce,
-    pub G: PaillierCiphertext, // Technically can be public but is only one per party
-    pub K: PaillierCiphertext, // Technically can be public but is only one per party
+    pub nu: Nonce,
+    pub G: Ciphertext, // Technically can be public but is only one per party
+    pub K: Ciphertext, // Technically can be public but is only one per party
```

Comment:
Comment totally unrelated to this PR. The choice to include G, K here makes me wonder (again) whether these values are appropriately included in the ZK proof. (Unless I'm misremembering what these objects are, aren't they part of the proof statement?)

Reply:
Hm, good question. `PiEnc` proves knowledge of the plaintext value of `K`, and `K` is used to construct it and to verify it and is put into the transcript to generate the challenge but is not formally part of the proof object (either in the code or as written in the proof itself, as far as I understand). `G` doesn't get used in round 1 but it's incorporated into a `PiLog` in round 2. I haven't checked to see how exactly that gets incorporated.